CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Stokes flow dataset
```

Comment:
Very minor. To be consistent with other changelog additions, can you please rewrite this to "Added Stokes flow dataset"?